src/button.jsx

```diff
@@ -3,7 +3,7 @@ import classNames from 'classnames';
 
 export default function Button(props) {
   return (
-    <button className={classNames('button', props.className)}>
+    <button {...props}>
```

Comment:
Looks good, but please keep the existing code for className otherwise we'll lose the class `button` that should always be applied :)

```suggestion
    <button {...props} className={classNames('button', props.className)}>
```